02_tools-and-packages/02_numpy.ipynb

```diff
@@ -137,9 +137,9 @@
     "## Create arrays manually\n",
     "\n",
     "* The \"manual\" syntax to create a numpy array is \n",
-    "  * `np.array([x, x])` for one dimensional arrays \n",
-    "  * and `np.array([[x, x], [x, x]])` for multidimensional arrays.\n",
-    "* Add to this, various functions like `np.ones()`, `np.eye()`, `np.arrange()`, `np.linspace()`, and many more create specific arrays which are often required in matrix computing."
+    "  * `np.array(a_list)` for one dimensional arrays \n",
+    "  * and `np.array([a_list, another_list])` for multidimensional arrays.\n",
```

Comment:
Could we switch back to this style? I guess it is more descriptive for students than `a_list`.

```suggestion
    "  * `np.array([x, x])` for one dimensional arrays\n",
    "  * and `np.array([[x, x], [x, x]])` for multidimensional arrays.\n",
```

Reply:
I was a bit confused by `[x, x]`. I was thinking it was a fixed number of arguments, but this is not the case. I must admit that `a_list` is also not very precise. Maybe you can use `[x_0, x_1, ..., x_n]`?